src/burnchains/db.rs

```diff
@@ -777,4 +777,262 @@ mod tests {
             panic!("EXPECTED to parse a stack stx op");
         }
     }
+
+    #[test]
```

Comment:
can tests go in files that have the word `test` in them? i think this is new best-practice style, even if it's not in style guide yet.

Reply:
I think we can aim to make these changes after 2.1, but I agree that this is a good practice.